### ISSUE TITLE
[cjson/libzen] [vcpkg_cmake_config_fixup] It doesn't fix up the ${_IMPORT_PREFIX}

### ISSUE THREAD

**Issue by PhoebeHui** [microsoft/vcpkg#20318]
We found vcpkg_cmake_config_fixup doesn't work for the following ports, It doesn't fix up the ${_IMPORT_PREFIX} in cmake config and targets file. cc @strega-nil 

- [ ] cjson
- [ ] libzen

eg:
cjson-debug.cmake:

```
#----------------------------------------------------------------
# Generated CMake target import file for configuration "Debug".
#----------------------------------------------------------------

# Commands may need to know the format version.
set(CMAKE_IMPORT_FILE_VERSION 1)

# Import target "cjson" for configuration "Debug"
set_property(TARGET cjson APPEND PROPERTY IMPORTED_CONFIGURATIONS DEBUG)
set_target_properties(cjson PROPERTIES
  IMPORTED_IMPLIB_DEBUG "E:/vcpkg/src/vcpkg/packages/cjson_x86-windows/debug/lib/cjson.lib"
  IMPORTED_LOCATION_DEBUG "E:/vcpkg/src/vcpkg/packages/cjson_x86-windows/debug/bin/cjson.dll"
  )

list(APPEND _IMPORT_CHECK_TARGETS cjson )
list(APPEND _IMPORT_CHECK_FILES_FOR_cjson "E:/vcpkg/src/vcpkg/packages/cjson_x86-windows/debug/lib/cjson.lib" "E:/vcpkg/src/vcpkg/packages/cjson_x86-windows/debug/bin/cjson.dll" )

# Commands beyond this point should not need to know the version.
set(CMAKE_IMPORT_FILE_VERSION)
  )
```

**Comment by strega-nil-ms** [comment]
Unfortunately this isn't a bug in vcpkg that we should fix, this is a bug in these ports' build systems; we need to fix the build systems instead (and probably add a check to vcpkg-tool)

**Comment by BillyONeal** [comment]
> Unfortunately this isn't a bug in vcpkg that we should fix, this is a bug in these ports' build systems; we need to fix the build systems instead (and probably add a check to vcpkg-tool)

@strega-nil @strega-nil-ms I think the issue fixed by #20380 where the wrong case of the variable was used is what this issue is about; is there something else you meant that vcpkg can't fix up?

**Comment by strega-nil-ms** [comment]
oh, apparently I misunderstood! thanks @BillyONeal
